tomster-player/.travis.yml

```diff
@@ -1,24 +0,0 @@
----
-language: node_js
-node_js:
-  - "8"
-
-sudo: false
-dist: trusty
-
-addons:
-  chrome: stable
-
-cache:
-  directories:
-    - $HOME/.npm
-
-env:
-  global:
-    # See https://git.io/vdao3 for details.
-    - JOBS=1
-
-script:
-  - npm run lint:hbs
-  - npm run lint:js
-  - npm test
```

Comment:
This isn't strictly related to the Node 8 topic and could be done in a separate PR 🤓

Reply:
Removed in favour of keeping the pr contents topic relevant, will remove the extra travis config in a following pr 👍